packages/react/src/components/Card/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+const ELEVATIONS = [0, 1, 2];
+
+const Card = ({ className, elevation, isSelected, ...props }) => (
+  <div
+    className={cx("Card", className, {
+      "is-selected": isSelected,
+      [`is-elevated--${elevation}`]: elevation,
+    })}
+    {...props}
+  />
+);
+
+Card.propTypes = {
+  className: PropTypes.string,
+  elevation: PropTypes.oneOf(ELEVATIONS),
+  isSelected: PropTypes.bool,
+};
+
+Card.defaultProps = {
+  className: undefined,
+  elevation: 0,
+  isSelected: false,
+};
+
+Card.ELEVATIONS = ELEVATIONS;
```

Comment:
Wouldn't it be cleaner to:

```
export {ELEVATIONS, Card}
export default Card 
```

Like this it makes me think I should/can do

```
import Card from ....
...
<>
   <Card elevation={Card.ELEVATIONS} />
<>
```

Which I'm not sure it's valid and can be confusing

Reply:
I think @obedparla what you want is actually achieved by the current implementation, and the suggestion would disable it

Reply:
You guys are right 😄